### ISSUE TITLE
Request: stagger the unsuspending of many tabs at once to avoid Chrome eating up all the CPU resources for minutes

### ISSUE THREAD

**Issue by jpbianchi** [deanoemcke/thegreatsuspender#854]
Please complete the following information when submitting a feature request or bug report.
 * Extension version: 7.0.109
 * Browser name & version: 72.0.3626.81
 * Operating system & version: osx 10.13.6

I have a few windows with more than 50 tabs, and when I unfreeze them, by accident or voluntarily, Chrome uses all my cores (12), runs them at 85% capacity, which freezes my computer for a few minutes.
It'd be great to pace unsuspending tabs when there's, say, more than 20.  Maybe this number could be a setting.
Thx

**Comment by Jollfye** [comment]
Also ran into this issue some time ago, unsuspension of all tabs is a nightmare for me, so I never ever use it, most of the time I have no more than 5-7 tabs unsuspended.

Plus extension should behave correctly when told to suspend all tabs while they're all in the process of being unsuspended, suspend them all back immediately, like if user changed mind. It happened to me, when all tabs were unsuspending and CPU + RAM was gone, I tried to press suspend all tabs in all windows (even forced option), nothing happened, my pc was done, I had to open task manager and end Chrome there, only then I was able to start new empty Chrome window and restore my tabs from session management, tho all tabs' titles were lost, which was extremely bad to me, which is why I wanna session management save them too..

**Comment by jpbianchi** [comment]
Wow, unfreeze 5-7 tabs only and your computer's gone!? I just tried and it's true that Chrome immediately took 95% of all cores, but it lasted only 10 seconds.  There's no real reason to unfreeze lots of tabs at once, my request comes from a bug I reported, which suddenly unsuspends all the tabs in a window due to a bug in chrome, which triggers hell on Earth.  
Due to that, the extension has been changed to do just what you (and I) want, ie being able to freeze all tabs while they're being unfrozen.  I think it's in version 165 from the top of my head.
